src/serac/physics/utilities/functional/functional.hpp

```diff
@@ -196,8 +205,7 @@ class Functional<test(trial), execution_policy> : public mfem::Operator {
    * and @a spatial_dim template parameter
    */
   template <int dim, typename lambda, typename qpt_data_type = void>
-  void AddBoundaryIntegral(Dimension<dim>, lambda&& integrand, mfem::Mesh& domain,
-                           QuadratureData<qpt_data_type>& data = dummy_qdata)
+  void AddBoundaryIntegral(Dimension<dim>, lambda&& integrand, mfem::Mesh& domain)
```

Comment:
For a future PR: there's some internal inconsistency where sometimes `lambda` gets passed and sometimes `lambda&&`; I don't think it makes a huge difference since functors are usually pretty lightweight, but since the lambda gets passed down a fairly long call chain maybe the latter is preferable.

Reply:
Good point, these should be made consistent. I'm not sure if an rvalue reference can be passed in to a cuda kernel, but we can find out.